source/_data/themes.yml

```diff
@@ -3180,3 +3180,16 @@
     - RhodesIsland
     - 明日方舟
     - 中文
+
+- name: Clean
+  description: A modfy clean blog theme with more feature
+  link: https://github.com/luswdev/site.git
```

Comment:
The link looks incorrect. It should be

https://github.com/luswdev/hexo-theme-clean

Reply:
I fixed it at lastest commit.